README.md

```diff
@@ -186,6 +186,80 @@ for metric in api.list_metrics(name=" "):
+a measurement with `hostname` and `company` tags.
+
+```
+  api = librato.connect(user, token)
+  api.submit_tagged("connections", 20, tags={'hostname': 'web-host-1', 'company': 'Librato'})
+```
+
+Multidimensional measurements can be submitted in batch mode using Queue's `add_tagged()` method, which also accepts an
+optional `tags` dictionary argument.
+
+```
+  api = librato.connect(user, token)
+  q   = api.new_queue()
+  q.add_tagged("connections", 30, tags={'hostname': 'web-host-1', 'company': 'Librato'})
+```
+
+Note that multidimensional measurements must be submitted using `submit_tagged()` or `add_tagged()`, regardless
+of whether or not a tag set has been specified. Any measurements submitted using `submit()` or `add()` will
+be directed to the traditional (untagged) Librato backend. Also, it is possible to add tagged and untagged
+measurements to a Queue instance, using `add_tagged()` and `add()`, respectively. Calling `submit()` will push
+the measurements to the respective backends (multidimensional or traditional).
+
+### Top-level tags
+
+The above examples showed how to attach tags to individual measurements. It is also possible to attach top-level tags
+to Connections and Queue instances, which effectively applies those tags to every measurement sent using the corresponding
+instance. The top-level tags will be combined with, and superceded, by any measurement-specific tags.
+
+The following example shows how to attach a `hostname` top-level tag to a Connection. The `connections` measurement will end
+up with both `hostname` and a `company` tags.
+
+```
+  api = librato.connect(user, token, tags={'hostname': 'web-host-1'})
+  api.submit_tagged("connections", 30, tags={'company': 'Librato'})
+```
+
+A Queue instance will inherit the top-level tags from the corresponding Connection object. In the following example,
+the Queue instance will inherit `hostname` as a top-level tag from the Connecton instance. The `connections` measurement
+will also end up with 'hostname' and 'company' tags.
+
+```
+  api = librato.connect(user, token, tags={'hostname': 'web-host-1'})
+  q = api.new_queue()
+  q.add_tagged("connections", 30, tags={'company': 'Librato'})
+```
+
+Top-level tag sets can be updated using `add_tags()` and replaced entirely using `set_tags()`, as shown below in the Connection
+example below. The Queue class follows an identical pattern.
+
+```
+  api = librato.connect(user, token, tags={'hostname': 'web-host-1'})
+  api.add_tags({'company': 'Librato'})        # Top-level tag set now has 'company' and 'hostname'
+  api.set_tags({'hostname': 'web-host-2'})    # Top-level tag set only has 'hostname'
+```
+
+### Querying tagged measurements
+Tagged measurements can be queried using the Connection `get_tagged()`. This method is much like `get()` but accepts a
+`tags_search` parameter, that specifies a tag query string. Refer to the main developer documentation for additional details
```

Comment:
Agreed. I believe the MD API docs are not officially out there yet. I am making a note to update the README once they get published.